.storybook/preview.js

```diff
@@ -32,3 +32,30 @@ const withThemeProvider = (Story, context) => {
 };
 
 export const decorators = [withThemeProvider];
+
+export const parameters = {
+  options: {
+    storySort: {
+      order: [
+        "Style",
+        ["Color", "Icons"],
+        "Components",
+        [
+          "Navigation",
```

Comment:
For headers on level 1 (`STYLE`, `COMPONENT`), the order should be predefined.
For the Intro and Style section, the order of levels 2 should be predefined.
For the library itself (inside `COMPONENTS`), it could be alphabetical order, however it makes sense to put `Navigation` at the beginning, to set `Input` and Selector next to each other, and also `Chart` and `Progress & loading` aside, as they are rather visual components.
At the end/bottom, it could be miscellaneous components.

![image](https://user-images.githubusercontent.com/28654802/109862089-4f0eb280-7c60-11eb-92ce-5d8aaa787ce2.png)

Reply:
Thank you for this explanation.